src/platform/forms-system/src/js/validation.js

```diff
@@ -40,13 +44,13 @@ const defaultMessages = {
 function getMessage(path, name, uiSchema, errorArgument) {
   let pathSpecificMessage;
   if (path === 'instance') {
-    pathSpecificMessage = _.get(['ui:errorMessages', name], uiSchema);
+    pathSpecificMessage = get(['ui:errorMessages', name], uiSchema);
   } else {
     const cleanPath = path
       .replace('instance.', '')
       .replace(/\[\d+\]/g, '.items');
-    pathSpecificMessage = _.get(
-      `${cleanPath}['ui:errorMessages'].${name}`,
+    pathSpecificMessage = get(
+      `${cleanPath}[ui:errorMessages].${name}`,
```

Comment:
Since it looks like `cleanPath` might be a property chain I think this is fine as-is.